lib/Importer/ONNX.cpp

```diff
@@ -502,6 +502,33 @@ std::unique_ptr<ONNXModelLoader> ONNXModelLoader::parse(const void *onnxModel,
 ONNXModelLoader::ONNXModelLoader(Function &F)
     : CommonOperatorLoader({}, {}, F) {}
 
+void ONNXModelLoader::checkInputs(onnx::GraphProto &net,
+                                  llvm::ArrayRef<const char *> names,
+                                  llvm::ArrayRef<Tensor *> tensors) {
+  for (size_t i = 0; i < names.size(); i++) {
+    // Look if a corresponding input exists.
+    for (int j = 0; j < net.input_size(); j++) {
+      const onnx::ValueInfoProto &valueInfo = net.input(j);
+      const std::string &inputName = valueInfo.name();
+
+      if (inputName == names[i]) {
```

Comment:
Related to the nest level as mentioned by @nadavrot, this `if` could be replaced by:
```
if (inputName != names[i]) {
  continue;
}
```
And then the current body of the `if` follows this new `if`, with one fewer nest level.

Reply:
OK, got it. Done.